lighthouse-core/gather/driver/network-monitor.js

```diff
@@ -162,7 +160,7 @@ class NetworkMonitor extends EventEmitter {
       const request = requests[i];
       if (request.finished) continue;
       if (requestFilter && !requestFilter(request)) continue;
-      if (NON_NETWORK_PROTOCOLS.includes(request.parsedURL.scheme)) continue;
+      if (URL.isNonNetworkProtocol(request.protocol)) continue;
```

Comment:
prefer the network request method when we have the complete request?

Reply:
yup good call thx.